_layouts/default.html

```diff
@@ -34,13 +50,10 @@
 			display: none !important;
 		}
 	</style>
-	<link rel="stylesheet" href="https://www.ft.com/__origami/service/build/v2/bundles/css?modules=o-normalise@^1.0.0,o-typography@^5.8.0,o-layout@^3.1.5,o-header-services@^3.2.9,o-footer-services@^2.0.3,o-syntax-highlight@^1.4.0,o-message@^3.1.0,o-table@^7.2.1&brand=internal" />
+	<link rel="stylesheet" href="https://www.ft.com/__origami/service/build/v2/bundles/css?modules=o-normalise@^2.0.0,o-typography@^6.0.0,o-layout@^4.0.0,o-header-services@^4.0.0,o-footer-services@^3.0.0,o-syntax-highlight@^3.0.0,o-message@^4.0.0,o-table@^8.0.0&brand=internal" />
```

Comment:
Use latest o-layout for sure
```suggestion
	<link rel="stylesheet" href="https://www.ft.com/__origami/service/build/v2/bundles/css?modules=o-normalise@^2.0.0,o-typography@^6.0.0,o-layout@^4.0.4,o-header-services@^4.0.0,o-footer-services@^3.0.0,o-syntax-highlight@^3.0.0,o-message@^4.0.0,o-table@^8.0.0&brand=internal" />
```